web_ui/src/pages/camera-page/components/action-buttons/accept-button.component.tsx

```diff
@@ -1,68 +1,87 @@
 // Copyright (C) 2022-2025 Intel Corporation
 // LIMITED EDGE SOFTWARE DISTRIBUTION LICENSE
 
+import { useState } from 'react';
+
 import { paths } from '@geti/core';
 import { Button } from '@geti/ui';
 import { groupBy, isEmpty } from 'lodash-es';
-import { useNavigate } from 'react-router-dom';
+import { NavigateFunction } from 'react-router-dom';
 
 import { DatasetIdentifier } from '../../../../core/projects/dataset.interface';
+import { NOTIFICATION_TYPE } from '../../../../notification/notification-toast/notification-type.enum';
+import { useNotification } from '../../../../notification/notification.component';
+import { TooltipWithDisableButton } from '../../../../shared/components/custom-tooltip/tooltip-with-disable-button';
 import { getIds } from '../../../../shared/utils';
 import { useDatasetMediaUpload } from '../../../project-details/components/project-dataset/hooks/dataset-media-upload';
 import { useCameraParams } from '../../hooks/camera-params.hook';
 import { useCameraStorage } from '../../hooks/use-camera-storage.hook';
 
 interface AcceptButtonProps {
     isDisabled?: boolean;
-    isPending?: boolean;
-    onPress?: () => void;
+    navigate: NavigateFunction;
 }
 
+export const insufficientStorageMessage =
+    'Your server is running low on disk space. Please contact customer support to find out possible solutions.';
```

Comment:
> Please contact customer support

Since we won't be supporting our saas product anymore and so don't have a customer support we should change these notifications to mention to look into the docs or issues online to find best practices to deal with data storage shortages.
I'm not sure if we have an up to date page about this on our documentation website though.

As an alternative maybe we can suggest to remove old projects or upgrade their hardware.

Reply:
Updated to `Your server is running low on disk space. Please free up space by removing old or unused projects, or consider upgrading your hardware to increase capacity.`